.github/mergify.yml

```diff
@@ -215,3 +216,22 @@ pull_request_rules:
     label:
       remove:
         - e2e-trigger
+
+- name: Trigger e2e workflow when two approvals are given
+  conditions:
+    - "#approved-reviews-by>=2"
+    - -files~=tests/.*$ # do not trigger on .py test files
```

Comment:
but this means if there's large functional change that includes tests, it will not run.  That doesn't seem quite right, either.

it would have to be something like changing the simple `*.py` check to be checking for python changes outside of that directory.  It'd be a more complicated regex, but doable I think.